classes/mender-sdimg.bbclass

```diff
@@ -167,15 +167,18 @@ EOF
     # Call WIC
     IMAGE_CMD_wic
 
+    wicimgname="${IMGDEPLOYDIR}/${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.wic"
```

Comment:
I'm using merge strategy for these branches, since they are 99% identical (actually 100% until now), so I'll merge as usual and revert the krogoth-incompatible patch specifically.